styles2.css

```diff
@@ -145,10 +149,297 @@ section.action {
 
 button {
   border: none;
+}
+
+button:hover {
+  background: #36b37f;
+  box-shadow: 0 8px 16px rgba(54, 179, 127, 0.24);
+  border-radius: 4px;
+}
+
+button:focus {
+  background: #008552;
+  border-radius: 4px;
+}
+
+button:enabled {
+  background: #36b37f;
+  border-radius: 4px;
+}
+
+section.headline > button {
   margin-top: 58px;
 }
 
-@media only screen and (min-width: 768px) {
+/*
+Second part of project
+*/
+
+section.recentWork {
+  width: 100vw;
+  height: auto;
+  padding: 64px 24px;
+  display: grid;
+  grid-template-columns: 1fr;
+  align-items: center;
+  gap: 58px;
+}
+
+section.recentWork > div.rHeader {
+  width: 90vw;
+  height: auto;
+  padding: 0 auto;
+}
+
+section.recentWork > div.rHeader > h1 {
+  margin: 0 auto;
+}
+
+h2 {
+  text-align: center;
+  font-weight: 600;
+  font-size: 24px;
+  line-height: 32px;
+  color: var(--letter-colorRC);
+  padding: 0 auto;
+}
+
+div.detail > div.rHeader > h2 {
+  margin: 0 auto;
+}
+
+.bar {
+  width: 48px;
+  height: 4px;
+  background-color: var(--dark-green);
+  border-radius: 24px;
+  margin: 0 auto;
+  margin-top: 24px;
+}
+
+section.recentWork > article {
+  width: 90vw;
+  border-radius: 8px;
+  background-color: var(--dark-white);
+  border: 1px solid #d0d9d4;
+}
+
+section.recentWork > article > div {
+  width: 100%;
+}
+
+.vide {
+  margin-top: 0;
+  width: 100%;
+  height: 45vh;
+}
+
+section.recentWork > article > div.detail {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-between;
+  padding: 16px;
+  gap: 16px;
+  background: #fff;
+  border-radius: 8px;
+}
+
+article.text > button {
+  margin-bottom: 0;
+}
+
+section.recentWork > article > div.detail > button {
+  margin-bottom: 8px;
+}
+
+/*
+section.recentWork > article > div.detail > ul
+*/
+
+ul.language {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  flex-wrap: wrap;
+  padding: 0;
+  gap: 8px;
+  width: 90%;
+}
+
+ul.language > li {
+  height: 32px;
+  padding: 8px 12px;
+  background-color: var(--dark-white);
+  border-radius: 4px;
+}
+
+.buttonCol > span {
+  padding: 12px;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  color: white;
+  letter-spacing: 0.03em;
+}
+
+ul.language > li > span {
+  font-weight: 600;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  color: var(--letter-colorRC);
+}
+
+.buttonCol {
+  background-color: var(--dark-green);
+  width: 159px;
+  height: 48px;
+  border-radius: 8px;
+  margin-left: 30%;
+  margin-right: 30%;
+}
```

Comment:
- [ ] As suggested by the previous reviewer, I think it would be better to reduce the space between ```"See project"``` and the button border to replicate the chosen [Figma design ](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48%3A988&t=Rsij71krcbUGyzAK-0) 😀

    _You may Kindly try to change the width of a button to ```width: 122px;``` as given by Figma 🙏 I hope this will solve this spacing issue :+1:_

### Your design currently
![image](https://user-images.githubusercontent.com/102757126/212474275-8ea238ad-3fa3-459f-aec4-b8fe7c96879c.png)

### Figma design
![image](https://user-images.githubusercontent.com/102757126/212474328-ce27677a-41ff-4809-be24-b84606a4d976.png)